tests/clone_test.go

```diff
@@ -840,3 +852,31 @@ var _ = Describe("[Serial]VirtualMachineClone Tests", Serial, func() {
 		})
 	})
 })
+
+func startCloneVM(virtClient kubecli.KubevirtClient, vm *virtv1.VirtualMachine) (*virtv1.VirtualMachine, error) {
+	patch, err := patch.New(patch.WithReplace("/spec/runStrategy", virtv1.RunStrategyAlways)).GeneratePayload()
+	if err != nil {
+		return nil, err
+	}
+
+	updatedVM, err := virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, patch, v1.PatchOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	return updatedVM, err
+}
+
+func stopCloneVM(virtClient kubecli.KubevirtClient, vm *virtv1.VirtualMachine) (*virtv1.VirtualMachine, error) {
+	patch, err := patch.New(patch.WithReplace("/spec/runStrategy", virtv1.RunStrategyHalted)).GeneratePayload()
```

Comment:
I think it is safer and better to use `WithAdd`.

Reply:
Are you sure? I think only replace would work here as /spec/runStrategy exists at that point

Reply:
Yes, I'm sure. `add` will override or add any existing value.
It is unsafe to use `replace` because you require it to be there, and that is assured only if it was added in the first place and not removed after (e.g. an update could have removed it).

In general, I would avoid using `replace` as it has no added value and raises questions like the above.

Reply:
Right. Addressed with some of the other comments
https://github.com/kubevirt/kubevirt/compare/65e089752d2e4db39fe6fc325dc2fba383c46cf0..961622903fd0e446efe859b11c654725e01a588d